SPECS/cloud-init-vmware-guestinfo/cloud-init-vmware-guestinfo.spec

```diff
@@ -0,0 +1,41 @@
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+Name:           cloud-init-vmware-guestinfo
+Version:        1.3.1
+Release:        1
+Summary:        A cloud-init datasource for VMware
+Group:          System/Management
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/vmware/cloud-init-vmware-guestinfo
+BuildArch:      noarch
+
+#Source0:  https://github.com/vmware/%{name}/archive/v%{version}.tar.gz
+Source0:       %{name}-%{version}.tar.gz
+BuildRequires: python3
+Requires:      cloud-init
+
+%description
+Provides a cloud-init datasource for pulling meta, user,
+and vendor data from VMware vSphere's GuestInfo interface.
+
+%prep
+%setup -q
+
+%build
+
+%install
+install -dm 0755 %{buildroot}%{_sysconfdir}/cloud/cloud.cfg.d
+install -m 0644 99-DataSourceVMwareGuestInfo.cfg %{buildroot}%{_sysconfdir}/cloud/cloud.cfg.d/99-DataSourceVMwareGuestInfo.cfg
+install -dm 0755 %{buildroot}%{python3_sitelib}/cloudinit/sources/
+install -m 0644 DataSourceVMwareGuestInfo.py %{buildroot}%{python3_sitelib}/cloudinit/sources/DataSourceVMwareGuestInfo.py
+
+%files
+%license LICENSE
+%config %{_sysconfdir}/cloud/cloud.cfg.d/99-DataSourceVMwareGuestInfo.cfg
+%{python3_sitelib}/cloudinit/sources/DataSourceVMwareGuestInfo.py
+
+%changelog
+* Thu Sep 17 2020 Mateusz Malisz <mamalisz@microsoft.com> 1.3.1-1
+- Initial CBL-Mariner packaging.
```

Comment:
>Initial CBL-Mariner packaging. [](start = 2, length = 30)

For our own specs please use the phrase `Original version for CBL-Mariner.`. See: [point the  21st from our guidelines](https://dev.azure.com/mariner-org/mariner/_wiki/wikis/mariner.wiki/125/Introduction-to-SPEC-Files).